server/models/Profile.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+
+const ProfileSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+    },
+    gender: {
+      type: String,
```

Comment:
Optional: You can look into using enum validators for the gender field

Ref: https://mongoosejs.com/docs/validation.html